src/components/dialogs/parameters/user-settings/user-settings-dialog.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { SyntheticEvent } from 'react';
+import { useDispatch } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { Alert, Dialog, Switch, Button, DialogActions, DialogContent, DialogTitle, Box, Theme } from '@mui/material';
+import { PARAM_DEVELOPER_MODE } from '../../../../utils/config-params';
+import { useParameterState } from '../parameters';
+import { selectEnableDeveloperMode } from '../../../../redux/actions';
+
+export interface UserSettingsDialogProps {
+    open: boolean;
+    onClose: () => void;
+}
+
+export const styles = {
+    parameterName: (theme: Theme) => ({
+        fontWeight: 'bold',
+        marginTop: theme.spacing(1),
+        flexGrow: 1,
+    }),
+    controlItem: {
+        flexGrow: 1,
+    },
+    parameterLine: {
+        display: 'flex',
+    },
+};
+
+/**
+ * Dialog to display user settings
+ * @param open Is the dialog open ?
+ * @param onClose Event to close the dialog
+ */
+export default function UserSettingsDialog({ open, onClose }: Readonly<UserSettingsDialogProps>) {
+    const dispatch = useDispatch();
+
+    const [enableDeveloperMode, handleChangeEnableDeveloperMode] = useParameterState(PARAM_DEVELOPER_MODE);
+
+    const handleSwichDeveloperMode = (enabled: boolean) => {
+        dispatch(selectEnableDeveloperMode(enabled));
```

Comment:
This is done by notification, do not do it here ! (see `updateParams` in app.jsx)